From 5fd048727908d0d0a95de94ed19f98a23fe4a77a Mon Sep 17 00:00:00 2001
From: Tim Waugh <twaugh@redhat.com>
Date: Fri, 3 Jul 2009 17:16:18 +0100
Subject: [PATCH 2/2] Map gutenprint filenames to the package name.

* cupshelpers/cupshelpers.py
(missingPackagesAndExecutables.pathcheck): Map gutenprint
filenames to the package name.
---
 cupshelpers/cupshelpers.py |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/cupshelpers/cupshelpers.py b/cupshelpers/cupshelpers.py
index febc0e1..b7e709a 100755
--- a/cupshelpers/cupshelpers.py
+++ b/cupshelpers/cupshelpers.py
@@ -841,8 +841,10 @@ def missingPackagesAndExecutables(ppd):
             # IJS servers (used by foomatic)
             'hpijs': 'hpijs',
             'ijsgutenprint.5.0': 'gutenprint',
+            'ijsgutenprint.5.2': 'gutenprint',
             # CUPS filters
             'rastertogutenprint.5.0': 'gutenprint-cups',
+            'rastertogutenprint.5.2': 'gutenprint-cups',
             'commandtoepson': 'gutenprint-cups',
             'commandtocanon': 'gutenprint-cups',
             }
-- 
1.6.3

